src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -81,6 +81,9 @@ public ModelStateDictionary ModelState
         [Activate]
         public IUrlHelper Url { get; set; }
 
+        [Activate]
+        public IValidationExcludeFiltersProvider ValidationFilterProvider { get; set; }
```

Comment:
ugh. I don't like this. This is a pretty obscure service to put front and center on the controller. We should introduce a facade for the validation system and put that here instead.